assets/scss/_footer.scss

```diff
@@ -0,0 +1,59 @@
+@use 'uswds-core' as *;
+
+$base-font-size: 1rem;
+$base-font-sm: ($base-font-size * 0.8125);
+
+.footer {
+  background-color: color('primary-lightest');
+}
+
+.container, .usa-identifier__container {
+  max-width: 75rem;
+  margin: {
+    left: auto;
+    right: auto
+  }
+  padding: {
+    left: 3.5rem;
+    right: 3.5rem;
+  }
+}
```

Comment:
Is there any reason we can't use a standard [design system grid](https://designsystem.digital.gov/utilities/layout-grid/) here, instead of recreating these styles?

Reply:
I switched to the design system grid: it looks like we prefer semantic classes over stylesheets, so that's what I went with.  
It's not my personal preference, so if I got that wrong, please let me know, I'm happy to do all styling in SASS.